### ISSUE TITLE
New room list: avatar online presence dot & public room icon lack tooltips

### ISSUE THREAD

**Issue by twassman** [element-hq/element-web#30442]
### Steps to reproduce

1. Enable new room list
2. Hover over a room's icon in the lower-right corner, e.g. online presence (online/away/offline) for DMs or the icon denoting public rooms

#### What did you expect?

An explanatory tooltip should display (cf. old room list):

<img width="269" height="210" alt="Image" src="https://github.com/user-attachments/assets/8245f1dc-8d28-47bd-a6c9-373a1a5448ae" />

#### What happened instead?

No such tooltip displays:

<img width="369" height="282" alt="Image" src="https://github.com/user-attachments/assets/85907860-b982-4e50-a54f-75243231d97b" />

### Browser information

Firefox 141.0b9

### URL for webapp

staging.element.io

### Application version

Element version: 1.11.108

### Will you send logs?

No
